metricbeat/docs/modules/oracle.asciidoc

```diff
@@ -85,17 +85,16 @@ metricbeat.modules:
   metricsets:
     - tablespace
   enabled: true
-  hosts: ["user/pass@0.0.0.0:1521/ORCLPDB1.localdomain"]
+  hosts: ['user="user" password="pass" connectString="0.0.0.0:1521/ORCLPDB1.localdomain"']
```

Comment:
What i understand is 

The old format is hosts: ["user/pass@0.0.0.0:1521/ORCLPDB1.localdomain"]

New format is 

hosts: ['user="user" password="pass" connectString="0.0.0.0:1521/ORCLPDB1.localdomain"']

Is this a breaking change then ?

Reply:
Your understanding is correct. We went with using the DSN format to configure the module, as discussed with @jsoriano over the TDD. I am not sure if we should consider this as a breaking change or an enhancement, please let me know if I should change the position of the changelog entry accordingly.

Reply:
With this new format, the existing users of sql module for oracle might be impacted. With this logic, to me, it is a breaking change. Let us discuss in the team meeting.

Reply:
I think we should continue supporting both formats if possible, even if it is only for backwards compatibility reasons. This is something we do in the postgresql module for example.

Reply:
I agree @yug-elastic . It won't be a breaking change if it is backward compatible .

Reply:
Kindly update the documentation to mention the support of both formats.

Also, please update the need of escaping the backslash character.

Reply:
Makes sense, thank you.
Updated the documentation ([link](https://github.com/elastic/beats/pull/31368/files#diff-bb326e2221aee29daf0c82e51a6006849f5f691086a48b26e97640696dbda392)). Please take a look.